### PR TITLE
resume: add page

### PR DESCRIPTION
**Pull request by SethFalco**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/resume.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# resume
+
+> CLI tool to easily setup a new resume.
+> More information: <https://github.com/jsonresume/resume-cli>.
+
+- Create a new `resume.json` file in your current working directory:
+
+`resume init`
+
+- Validate your `resume.json` against our schema tests to ensure it complies with the standard:
```
Comment (CleanMachine1) [inline pages/common/resume.md:10]:
```suggestion
- Validate a `resume.json` against schema tests to ensure it complies with the standard:
```

I still don't like how its worded, maybe someone else can make this better.

File: pages/common/resume.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`resume validate`
+
+- Export your resume locally in a stylized HTML or PDF format:
```
Comment (CleanMachine1) [inline pages/common/resume.md:14]:
```suggestion
- Export a resume locally in a stylized HTML or PDF format:
```

File: pages/common/resume.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`resume export {{path/to/output}}`
+
+- Start a web server that serves your local `resume.json`:
```
Comment (CleanMachine1) [inline pages/common/resume.md:18]:
```suggestion
- Start a web server that serves a local `resume.json`:
```

File: pages/common/resume.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# resume
+
+> CLI tool to easily setup a new resume.
+> More information: <https://github.com/jsonresume/resume-cli>.
+
+- Create a new `resume.json` file in your current working directory:
```
Comment (CleanMachine1) [inline pages/common/resume.md:6]:
```suggestion
- Create a new `resume.json` file in the current working directory:
```

**Review by CleanMachine1 — CHANGES_REQUESTED** [review]

File: pages/common/resume.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`resume init`
+
+- Validate a `resume.json` against schema tests to ensure it complies with the standard:
+
+`resume validate`
+
+- Export a resume locally in a stylized HTML or PDF format:
+
+`resume export {{path/to/output}}`
```
Comment (marchersimon) [inline pages/common/resume.md:16]: Does the format depend on the file extention?
If so, `.../output.pdf|.html` looks weird, but maybe `path/to/html_or_pdf`?

Comment (SethFalco) [reply pages/common/resume.md:16]: Sounds fair, I think that could be nice.

```suggestion
`resume export {{path/to/html_or_pdf}}`
```

**Review by marchersimon — APPROVED** [review]
Looks good 👍🏽

**Review by einverne — APPROVED** [review]
It looks good. And I'm great to know this useful command.

**Review by CleanMachine1 — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**